packages/vector_graphics_compiler/lib/src/svg/parser.dart

```diff
@@ -558,6 +559,12 @@ class _SvgGroupTuple {
   final ParentNode drawable;
 }
 
+/// Parse an SVG to the initial Node tree.
+@visibleForTesting
+Future<Node> parseToNodeTree(String source) {
```

Comment:
How is this as a compromise? I can still make assertions on the Node structure, but I don't need to manually construct it.